changelog.d/10108.feature

```diff
@@ -0,0 +1 @@
+Implement config option sso.update_profile_information to keep a user's profile fields in sync with the values from the SSO ticket (currently display_name).
```

Comment:
```suggestion
Implement config option `sso.update_profile_information` to sync SSO users' profile information with the identity provider each time they login. Currently only displayname is supported.
```

As "SSO ticket" might not be a term most sysadmins are immediately familiar with.